kentikapi/credentials.go

```diff
@@ -4,31 +4,55 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 	"path/filepath"
 )
 
+type Profile struct {
+	AuthEmail string `json:"email"`
+	AuthToken string `json:"api-key"`
+}
+
+const DefaultProfile = "default"
+
+// Credentials lookup order (assuming profile name is "default"):
```

Comment:
Please put it into GetCredentials docstring, so that it is rendered into pkg.go.dev.

Reply:
Done